include/SDL3/SDL_main.h

```diff
@@ -94,6 +94,16 @@
         /* We need to export SDL_main so it can be launched from Java */
         #define SDLMAIN_DECLSPEC    SDL_DECLSPEC
 
+    #elif defined(SDL_PLATFORM_EMSCRIPTEN)
+        /* On Emscripten, SDL provides a main function that converts URL
+           parameters that start with "SDL_" to environment variables, so
+           they can be used as SDL hints, etc.
+
+           This is 100% optional, so if you don't want this to happen, you may
+           define SDL_MAIN_HANDLED
+         */
+        #define SDL_MAIN_AVAILABLE
```

Comment:
With this change in `include/SDL3/SDL_main.h`, I don't get any console output in the tests.

Visiting http://localhost:8080/testplatform.html should print text to the console, but nothing is printed there, or shown in the output textarea.